src/PipeDream.js

```diff
@@ -1,13 +1,18 @@
-import defaultStore from './defaultStore/index.js';
+import defaultStore from './defaultStore/index.js'
+const mergeJSON = require('deepmerge')
 
 export default class PipeDream {
     constructor(options) {
-        this.options = options
+        this.options = mergeJSON(this.defaultOptions(), options)        
```

Comment:
This is nice! Doesn't cross my mind to use deepmerge here.

Reply:
👍